grids/grids.css

```diff
@@ -0,0 +1,33 @@
+.container-grid {
+    width: 50%;
+}
+
+/* for multiple items in grid, uncomment the two lines below */
+.two-item-grid {
+    display: grid;
+    grid-gap: 10px;
+    /* grid-template-columns: repeat(2, 1fr); */
+    /* grid-template-rows: 0.1fr 1fr; */
+    grid-template-areas:
+        " sally tom"
+}
+
+#sally {
+    grid-area: sally;
+}
+
+#tom {
+    grid-area: tom;
+}
+
+/* Three Grid items */
+
+/* for multiple items in grid, uncomment the line below */
+.three-item-grid {
+    display: grid;
+    grid-gap: 10px;
+    grid-template-columns: repeat(3, 1fr);
+    /* grid-template-rows: 0.1fr 1fr; */
+    grid-template-areas:
+        " kelly nick josh"
```

Comment:
Sorry if I did not get the reference. Neutral naming like item1 item2 item3 is preferrable. And these are supposed to be overridden by the end user not in the library

Reply:
item1, item2 are too generic, especially for grid items. These names exist everywhere in our code. Therefore, I prefer to give specific IDs for grid items, so while laying them out in grid template areas, it become easier.

Reply:
Changed anyways!